traffic_portal/app/src/common/modules/form/user/FormUserController.js

```diff
@@ -50,7 +50,7 @@ var FormUserController = function(user, $scope, $location, formUtils, stringUtil
     $scope.labelize = stringUtils.labelize;
 
     $scope.viewDeliveryServices = function() {
-        $location.path($location.path() + '/delivery-services');
+        $location.path('/tenants/' + userModel.user.tenantId + '/delivery-services');
```

Comment:
fyi for whoever reviews this. i was the one that suggested when viewing a user's delivery services, it's probably easiest to simply send them to view the delivery services of the user's tenant instead.

this does mean, however, that we are abandoning a view (users/id/deliveryservices) that should probably be cleaned up.

Reply:
Here is a PR where @ocket8888 removed a view. In this case he removed the ability to view cachegroups for a parameter - https://github.com/apache/trafficcontrol/pull/3636